commands/deploy_command.go

```diff
@@ -85,17 +85,17 @@ func (c *DeployCommand) GetPluginCommand() plugin.Command {
    Perform action on an active deploy operation
    cf deploy -i OPERATION_ID -a ACTION [-u URL]`,
 			Options: map[string]string{
-				extDescriptorsOpt:                                  "Extension descriptors",
```

Comment:
What happened with the formatting?

Reply:
It is autogenerated from visual studio code and its extensions :/